### PR TITLE
Test if containers define the pre-stop hook for lifecycle management

### PR DESCRIPTION
**Pull request by HamadiSE**
Test if containers define the pre-stop hook for lifecycle management
update Dockerfile

Signed-off-by: salah hamadi <shamadi@redhat.com

### CODE REVIEW

File: pkg/tnf/handlers/shutdown/shutdown.gotemplate [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{{- range .spec.containers -}}
```
Comment (jun-rh) [inline pkg/tnf/handlers/shutdown/shutdown.gotemplate:1]: The point of this is just to make generic test return something more readable than obscure patterns?

Comment (jun-rh) [reply pkg/tnf/handlers/shutdown/shutdown.gotemplate:1]: Get it now, it also does a loop. Nice.

**Review by jun-rh — APPROVED** [review]

File: pkg/tnf/handlers/shutdown/shutdown.gotemplate (hunk 1 above)
Comment (HamadiSE) [reply pkg/tnf/handlers/shutdown/shutdown.gotemplate:1]: Yes exactly. otherwise, it will return the command defined in the output or some "<no value>".

Comment (edcdavid) [reply pkg/tnf/handlers/shutdown/shutdown.gotemplate:1]: This is nice

**Review by edcdavid — APPROVED** [review]
I see the test was failing because the template was not copied in the container

**Pull request merged**